src/components/allButtons.sc.js

```diff
@@ -1,5 +1,6 @@
 import styled, { css } from "styled-components";
 import { zeeguuOrange, lightOrange, lightGrey } from "./colors";
+import SearchIcon from "@mui/icons-material/Search";
```

Comment:
If I'm not wrong, a default import can be given any name we like. So we could import `MUISearchIcon` and export from this file simply a `SearchIcon`? It would be more in line with all the other buttons we're exporting from this file.